rtfm/source/_pages/build/prod.rst

```diff
@@ -0,0 +1,78 @@
+.. _BuildProd:
+
+Production build
+================
+
+.. code-block:: bash
+
+    yarn clean:dist && yarn build:prod
+
+Production build will essentially:
+
+* enable Webpack's `production mode`_
+* do not produce source maps for CSS
+* do minimize assets
+
+.. note::
+    By default the production build **will** produce source-maps for JS; this is done to
```

Comment:
with the commit https://github.com/welaika/wordless/pull/261/commits/6b95f16c66fc6541d5a955a20bcedf7abee63dc0
we need to change this part

Reply:
Updating the doc to

```rst
.. note::
    By default the production build won't produce source-maps for JS.

    You can easily change this behaviour updating `` const needSourceMap = (env.DEBUG == 'true');``
    to ``  const needSourceMap = true;`` in ``webpack.env.js``
```